applications/endoscopy_tool_tracking/cpp/CMakeLists.txt

```diff
@@ -156,29 +156,10 @@ endif()
 # Install application and dependencies into the install/ directory for packaging
 install(
   TARGETS endoscopy_tool_tracking
-  DESTINATION endoscopy_tool_tracking_cpp
-  COMPONENT "holohub-apps"
+  DESTINATION endoscopy_tool_tracking/cpp
```

Comment:
Following discussion with @jjomier @mocsharp , we are internally discussing realignment to produce a new install tree per build environment. I.e., suggest the following commands:
```
./dev_container build_and_install endoscopy_tool_tracking
./dev_container build_and_Install multiai_ultrasound
```
May generate one install tree per __app build environment__:
```
install/
├── endoscopy_tool_tracking
  ├── bin
  ├── lib
  ├── python
...
├── multiai_ultrasound
  ├── bin
  ├── lib
  ├── python
```
That reflection of the build environment and installation standard practices may reduce effort for aligning subsequent packaging interests.

Reply:
Seems that we are good with the current installation tree.